templates/cloudwatch.yml.j2

```diff
@@ -1,3 +1,4 @@
 ---
 region: {{ sansible_prometheus_cloudwatch_exporter.config.region }}
-metrics: {{ sansible_prometheus_cloudwatch_exporter.config.metrics }}
+metrics: 
+{{ sansible_prometheus_cloudwatch_exporter.config.metrics }}
```

Comment:
Are you sure there is to be a line break?

Reply:
Yes, it's to cope with Jinja formatting being rubbish (again).